Publisher/Sources/Services/DefaultAblyPublisherService.swift

```diff
@@ -18,13 +18,15 @@ class DefaultAblyPublisherService: AblyPublisherService {
 
     private func setup() {
         client.connection.on { [weak self] stateChange in
-            guard let current = stateChange?.current,
-                  let self = self
-            else { return }
-            logger.debug("Connection to Ably changed. New state: \(current)", source: "DefaultAblyPublisherService")
+            guard let self = self,
+                  let receivedConnectionState = stateChange?.current.toConnectionState() else {
+                return
+            }
+            
+            logger.debug("Connection to Ably changed. New state: \(receivedConnectionState)", source: "DefaultAblyPublisherService")
```

Comment:
> `source: "DefaultAblyPublisherService"`

I'm wondering if there's any `#class` literal like `#function`, `#file`, etc. If so, then I think we can use it instead of the literal string, it's safer. I usually forgot to change strings like this when the class is renamed 😄 

Maybe something like `String(DefaultAblyPublisherService.self)` works too, I guess there's something to get the name of the class.

EDIT: or extracting the name from the `#file` literal.

Reply:
I created [issue](https://github.com/ably/ably-asset-tracking-cocoa/issues/127) for this, because it is not the only place where it has to be changed :)